### ISSUE TITLE
Patreon Integration (stage 2)

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#1554]
We've now got our first Patreon subscribers (thanks to them all). 

It is  going to take some work to keep growing our subscriber base and to be able to cover all of the costs of running and growing the project. I'm putting forth the following plan, some of which we can implement immediately.

## Additional linkes on the website

- [ ] a 'support open publishing' link on the menubar
- [ ] a temporary funding drive banner (1 month) at top of every page, urging people to sign up
- [ ] a 'support' message on every lesson just above 'About the authors' In a yellow warning or green message box with Image (https://www.flickr.com/photos/britishlibrary/11055332325/ or https://www.flickr.com/photos/britishlibrary/11254363913/in/album-72157638906393085/). Found this lesson useful? Join the movement to build more and better opportunities for free technical education. Become a patron today.

## Targeted messaging for next six months

Places we will focus on in particular to advertise our Patreon page and what we plan to do with the support.

| Month         |  Target Audience             | What are they? |
| ------------- |:----------------------------:| --------------:|
| Dec - early   | French Patreon added and pushed w French speakers | French speaking DH people | 
| Dec - late    | none. It's Christmas         | n/a            |
| Jan - early   | none. Everyone is poor now   | n/a            |
| Jan - late    | Spanish Patreon added and pushed w Spanish speakers | Spanish speaking DH people |
| Feb - early   | Humanist Discussion Group | DH List Serv   |
| Mar - early   | Our own colleagues & friends |  people we know who might be willing to chip in a few dollars (whole team push) |
| Mar - late    | Library professionals | people interested in the collections end of DH |
| Apr - early   | 5 different H-Net Listservs | historians w specific interests we think are relevant to us|

## Specific Tasks:

- [x] Cover image could be more colourful.
- [x] Very text heavy.
- [ ] not clear the call to action is compelling enough (why do we need the money?)
- [x] French isn't really visible. Bilingual page isn't really working. (tweaks made to make French more visible)
- [ ] nothing to come back for (should we transfer blogging to Patreon to get eyes moving that way?)

Currently working to add French patreon content and promote with French speakers.

**Comment by acrymble** [comment]
Please comment if you have any feedback. I'll work on integrating some of the above over the next month. I hope nothing here is controversial.

**Comment by drjwbaker** [comment]
Pinging @programminghistorian/technical-team as some of these actions require their input on feasibility wrt the site infrastructure and their capacity to implement changes given their existing workloads.

**Comment by walshbr** [comment]
From a technical / UX perspective -

1. a 'support open publishing' link on the menubar - I think this is essentially what the "support us" link in the menu bar is already doing. I'd be reluctant to add another menu bar link to the already crowded header menu that duplicates something that is already done.
2.  a temporary funding drive banner (1 month) at top of every page, urging people to sign up - this is more technically involved and would need to be added to the tech team workflow. We're working up strategies based on the recent tech call for feature requests like these, but it would need to be added into the queue as it would take time (it'd require styling beyond just throwing text there).
3.  "a 'support' message on every lesson just above 'About the authors' In a yellow warning or green message box with Image (https://www.flickr.com/photos/britishlibrary/11055332325/ or https://www.flickr.com/photos/britishlibrary/11254363913/in/album-72157638906393085/). Found this lesson useful? Join the movement to build more and better opportunities for free technical education. Become a patron today." - this seems fine enough. It would just be an include that would need to be translated and stuck at the bottom of the lesson templates. It'd essentially just be a mirror of the "support us" link at the top.

**Comment by acrymble** [comment]
For anyone interested, I found this video by Patreon useful for thinking about building supporters https://www.youtube.com/watch?v=p1RnuFWXU_k

I think one of the key hurdles to get over is any embarassment about asking people to contribute to your project. If you ask with  a whisper in a footnote,  you won't get people's attention. So there's a balance to be struck, for sure, but being proud about what you can do with support is a good approach, I think.

**Comment by acrymble** [comment]
In  terms of what we already have, yes we have a 'support us' page, but people aren't visiting it. Of  65,000 visitors this week, only 22 loaded the 'support us' page. That's 1 every 3,000. So  it's a start, but let's keep thinking about how and why people should support us. The work we do has value.

<img width="1034" alt="Screenshot 2019-11-19 at 18 33 20" src="https://user-images.githubusercontent.com/646647/69175294-63767c80-0afb-11ea-8242-7eaabca7c93b.png">

**Comment by walshbr** [comment]
So are you just suggesting we rename "Support us" to "Support open publishing"? That seems fine if you want - I was just trying to point out that the one idea was sort of already being attempted. Thanks for the analytics - always appreciate how you're on top of those.

**Comment by walshbr** [comment]
I think the combination of passive representation, timed campaigns, and targeted advertising all seems good.

**Comment by mdlincoln** [comment]
Adding in an "alert" box to all the lessons (or even the lessons + all our other pages, homepages etc.) would be a pretty light lift since we already have similar alert boxes in our CSS (we can add a special snazzy color to them, of course). If we can get a sample of language and a decision on where to put them, then @programminghistorian/technical-team can get some folks working on it.

**Comment by drjwbaker** [comment]
Can we get a mock-up with lorem ipsum text as a placeholder?

**Comment by mdlincoln** [comment]
fine but we still need part 2 of my request - a decision on where to put them

**Comment by jenniferisasi** [comment]
One idea: 
-  add a small temporary banner above language banners on the front page

But! I just realized we are missing an opportunity for $$ in this section upon entering the site: 

> ## Contribute
> Write a lesson, join our team of reviewers, provide feedback. We're always keen to hear from our readers!
> 

I think we need to add the support us link there too, right?

**Comment by mdlincoln** [comment]
Can we move discussion about the particulars of new alerts/banners over to #1563?

**Comment by acrymble** [comment]
I'm also going to adjust our "goals" for monthly income to add a lower goal that makes it feel like we're closer (and therefore just need a bit more support). I will set this at 100/mo.  Affects Patreon  page only.

**Comment by acrymble** [comment]
French integration into Patreon:

@fdlaramee would you please help me to translate the following bits of our patreon page?:

* $100 per month will allow us to improve the sustainability of new lessons through careful professional copyediting (on top of the hard work of our editors). This will give learners and educators extra confidence in our lessons.

* Programming Historian is an international volunteer-driven project administered by ProgHist Limited, a not-for-profit company limited by guarantee registered in England (12192946). Our publications are published by the Editorial Board of the Programming Historian.

We seek to change the world through high quality open access learning.

We believe that everyone should have access to free education and the tools they need to master technology. We are a leading open access not-for-profit publisher specializing in beginner-friendly technology tutorials that allow learners to build skills at their own pace. We publish in English, Spanish, and French, and each year reach more than 1-million people around the world.

Your contribution enables us to maintain and update published lessons, to devote extra time to work with our authors so they produce clear and accessible instructions, and to expand our multi-lingual and culturally sensitive approach to learning and teaching.

For getting us off the ground, the first person to join a tier will be recognized on our sponsorship page. We really appreciate you.

Thank you for supporting our project.

* The Editorial Board of the Programming Historian thank you for your generous contribution to our work. You are amazing!

---

@spapastamkou @grandjeanmartin @fdlaramee could you also draft 4-5 tweets I could use to promote Patreon to French audiences via our twitter account? I will take care of integrating them. Something like "support digital humanities open access French resources etc etc etc..."

**Comment by acrymble** [comment]
Patreon have a button we can embed in our site. Do we like it?
![Screen Shot 2019-12-02 at 10 55 35](https://user-images.githubusercontent.com/646647/69954015-55a8fa00-14f2-11ea-8686-304aa42fd592.png)

**Comment by mdlincoln** [comment]
@acrymble does Patron offer the button in SVG format? That would be ideal for the best display - if not, then PNG is ok.

**Comment by acrymble** [comment]
It's just a javascript integration actually. I screenshotted it for this.

<a href="https://www.patreon.com/bePatron?u=11816614" data-patreon-widget-type="become-patron-button">Become a Patron!</a><script async src="https://c6.patreon.com/becomePatronButton.bundle.js"></script>

**Comment by fdlaramee** [comment]
@acrymble Here are the translations you requested:

    $100 per month will allow us to improve the sustainability of new lessons through careful professional copyediting (on top of the hard work of our editors). This will give learners and educators extra confidence in our lessons.

--> 100$ par mois nous permettront d'assurer la pérennité de nos nouvelles leçons grâce à une correction d'épreuves professionnelle (qui viendra s'ajouter au travail acharné de nos rédacteurs). Cela inspirera une confiance accrue dans nos leçons auprès des apprenants et des éducateurs.

    Programming Historian is an international volunteer-driven project administered by ProgHist Limited, a not-for-profit company limited by guarantee registered in England (12192946). Our publications are published by the Editorial Board of the Programming Historian.

--> Le Programming Historian est un projet international dirigé par des bénévoles et administré par ProgHist Limited, une corporation à but non lucratif enregistrée en Angleterre (12192946). Nos publications sont publiées par le Comité de rédaction du Programming Historian.


We seek to change the world through high quality open access learning.

--> Nous voulons changer le monde par l'apprentissage de haut niveau en libre accès.

We believe that everyone should have access to free education and the tools they need to master technology. We are a leading open access not-for-profit publisher specializing in beginner-friendly technology tutorials that allow learners to build skills at their own pace. We publish in English, Spanish, and French, and each year reach more than 1-million people around the world.

--> Nous croyons en l'accès universel et gratuit à la formation et aux outils nécessaires pour maîtriser la technologie. Nous sommes un chef de file dans le domaine de l'édition à but non lucratif en libre accès, spécialisé dans les leçons accessibles aux débutants qui permettent aux apprenants de bâtir leurs compétences techniques à leur propre rythme. Nous publions en anglais, en espagnol et en français et nous rejoignons plus d'un million de personnes de partout dans le monde à chaque année.

Your contribution enables us to maintain and update published lessons, to devote extra time to work with our authors so they produce clear and accessible instructions, and to expand our multi-lingual and culturally sensitive approach to learning and teaching.

--> Votre contribution nous permet de conserver et de mettre à jour les leçons publiées, de consacrer du temps additionnel à nos auteurs et autrices pour qu'ils et elles puissent produire des leçons claires et accessibles, et de développer notre approche d'apprentissage et d'enseignement multilingue et sensible aux différences culturelles.

For getting us off the ground, the first person to join a tier will be recognized on our sponsorship page. We really appreciate you.

--> En reconnaissance de votre contribution à notre démarrage, la première personne qui s'engagera à chacun des niveaux de contribution recevra des remerciements spéciaux sur notre page des contributeurs. Nous vous apprécions vraiment.

Thank you for supporting our project.

--> Merci de soutenir notre projet.

    The Editorial Board of the Programming Historian thank you for your generous contribution to our work. You are amazing!

--> Le Comité de rédaction du Programming Historian vous remercie de votre généreuse contribution à notre travail. Vous êtes fantastique!

**Comment by fdlaramee** [comment]
@acrymble Here are a couple of draft tweets:

Le Programming Historian en français publie des leçons pratiques sur la technologie et sur tous les aspects des humanités numériques. Soutenez-nous sur Patreon au http://...

Soutenez le développement de ressources en français dans le domaine des humanités numériques en contribuant au Patreon du Programming Historian: http://...

Vous avez acquis des compétences en humanités numériques à l'aide des leçons publiées par le Programming Historian? Soutenez nos efforts (y compris celles visant le développement de ressources en français) sur Patreon: http://...

Afin de poursuivre le développement de ressources en français pour l'apprentissage et l'enseignement des humanités numériques, le Programming Historian a besoin de votre soutien. Contribuez à notre Patreon: http://...

**Comment by acrymble** [comment]
Thanks @fdlaramee

**Comment by acrymble** [comment]
I did a bit of a focus group on patreon and got the following feedback on our patreon page.

- Cover image could be more colourful.
- Very text heavy.
- not clear the call to a ction is compelling enough (why do we need the money?)
- French isn't really visible. Bilingual page isn't really working.
- nothing to come back for  (should we transfer blogging to Patreon to get eyes moving that way?)

I will do some work to try to address some of these things and see what effect changes have on conversion rates (we've got a 0% conversion rate the last two weeks, on 100 visitors. Update: we have had a few new subscribers today.)
